doc/API.md

```diff
@@ -129,6 +129,7 @@ JitsiMeetJS.setLogLevel(JitsiMeetJS.logLevels.ERROR);
         - CHAT_ERROR - chat error happened
         - FOCUS_DISCONNECTED - focus error happened
         - FOCUS_DISCONNECTED - focus left the conference
+        - CONFERENCE_MAX_USERS - The maximum users limit is already reached
```

Comment:
Please also have a look for other places where you wrote "limit is reached" or "have been reached"(should be "has been reached")